quadriga/src/main/java/edu/asu/diging/quadriga/core/exceptions/CollectionNotFoundException.java

```diff
@@ -0,0 +1,18 @@
+package edu.asu.diging.quadriga.core.exceptions;
+
+public class CollectionNotFoundException extends Exception {
```

Comment:
Let's override all constructors, especially the one that takes an exception as parameter is important to keep the stack trace.